src/ui/public/vislib/styles/_tooltip.less

```diff
@@ -57,20 +57,20 @@
     margin-bottom: 0;
   }
 
-  &-icon {
+  + * {
```

Comment:
I hesitate to dig into this component, since I'm not sure of its scope. I'll tackle this guy separately.